### PR TITLE
[UI] - Minor test fixes

### PR DESCRIPTION
**Pull request by adarshdubey-star**
Minor test fix -[ Rhcloud insights] 

**Test results**
```
pytest tests/foreman/ui/test_rhcloud_insights.py -k test_host_details_page
=============================================================================================== test session starts ===============================================================================================
platform linux -- Python 3.8.6, pytest-7.1.2, pluggy-1.0.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/addubey/work/robottelo, configfile: pyproject.toml
plugins: services-2.2.1, mock-3.7.0, forked-1.3.0, cov-3.0.0, xdist-2.5.0, ibutsu-2.2.2, reportportal-5.1.2
collected 10 items / 9 deselected / 1 selected                                                                                                                                                                    

tests/foreman/ui/test_rhcloud_insights.py .                                                                                                                                                                 [100%]

============================================================================ 1 passed, 9 deselected, 3 warnings in 1882.10s (0:31:22) =============================================================================

```

### CODE REVIEW

**Comment by adarshdubey-star** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_rhcloud_insights.py -k 'test_host_details_page'

**Comment by devendra104** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_rhcloud_insights.py -k 'test_host_details_page'

File: tests/foreman/ui/test_rhcloud_insights.py [hunk 1]
```diff
@@ -287,6 +286,7 @@ def test_host_details_page(
             silent_failure=True,
             handle_exception=True,
         )
+        session.cloudinsights.sync_hits()
```
Comment (jameerpathan111) [inline tests/foreman/ui/test_rhcloud_insights.py:289]: We have `wait_for` before this line, which waits for sync task to complete.
So it doesn't look like the right fix to me.

Comment (adarshdubey-star) [reply tests/foreman/ui/test_rhcloud_insights.py:289]: Somehow the remediations are not following after the full inventory sync which is resulting in an assertion failure
for `assert int(result['Recommendations']) > 0` hence I am trying to sync recommendations after the complete inventory sync.

Comment (adarshdubey-star) [reply tests/foreman/ui/test_rhcloud_insights.py:278]: so what I am seeing is @jameerpathan111 removal of the wait_for function is not impacting the sync_hits for which we had the timeout hence so removing it wont be a issue and test is passing as well.

**Comment by adarshdubey-star** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_rhcloud_insights.py -k 'test_host_details_page'

File: tests/foreman/ui/test_rhcloud_insights.py (hunk 1 above)
Comment (jameerpathan111) [reply tests/foreman/ui/test_rhcloud_insights.py:278]: yeah if sync is instantaneous it shouldn't be an issue but if it's not then the test will fail.
But we can give it a try and see how it goes.

**Review by jameerpathan111 — APPROVED** [review]

File: tests/foreman/ui/test_rhcloud_insights.py (hunk 1 above)
Comment (adarshdubey-star) [reply tests/foreman/ui/test_rhcloud_insights.py:278]: Passed on PRT as well!

**Review by latran — APPROVED** [review]

**Review by pondrejk — APPROVED** [review]

**Pull request merged**